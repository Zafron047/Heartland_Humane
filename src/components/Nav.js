import React from 'react';
import { Link } from 'react-router-dom';
import '../styles/Detail.css';

const Nav = () => (
  <nav className="nav">
    <Link to="/">
      <div className="Back-Button">&#60;</div>
    </Link>
  </nav>
);

export default Nav;
