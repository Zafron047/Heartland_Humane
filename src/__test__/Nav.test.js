import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from '../components/Nav';

describe('Nav', () => {
  it('renders Nav component correctly', () => {
    const { container } = render(
      <Router>
        <Nav />
      </Router>,
    );

    expect(container).toMatchSnapshot();
  });
});
