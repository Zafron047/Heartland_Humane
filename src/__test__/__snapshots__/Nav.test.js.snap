// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders Nav component correctly 1`] = `
<div>
  <nav
    class="nav"
  >
    <a
      href="/"
    >
      <div
        class="Back-Button"
      >
        &lt;
      </div>
    </a>
  </nav>
</div>
`;
